src/compiler/scala/tools/nsc/ast/parser/Scanners.scala

```diff
@@ -912,7 +912,7 @@ trait Scanners extends ScannersCommon {
         }
       } else if (ch == '$') {
         nextRawChar()
-        if (ch == '$') {
+        if (ch == '$' || (currentRun.isScala3 && ch == '"')) {
```

Comment:
IMO we should allow this in 2.13, it doesn't break any existing code.

Reply:
That's still in pre-SIP state: https://contributors.scala-lang.org/t/pre-sip-mini-sip-escapes-in-interpolations/2076/

Reply:
Yeah, but it's in Scala 3, so 🤷‍♂️

Reply:
Then it probably also needs the spec changes in https://github.com/scala/scala/pull/6953